packages/react-tinacms-inline/src/inline-form.tsx

```diff
@@ -94,10 +94,9 @@ export function InlineForm({ form, children }: InlineFormProps) {
               <>
                 {typeof children !== 'function'
                   ? children
-                  : children({
-                      ...formProps,
-                      ...inlineFormState,
-                    })}
+                  : //
+                    // @ts-ignore
+                    children({ ...formProps, ...inlineFormState })}
```

Comment:
This one is messy.  I'm not quite sure what the issue is here other than we're not typing `formProps` and `inlineFormState` well enough and TS is assuming we're leaving out required parts of `InlineFormRenderOptions`:

```
Argument of type '{ form: Form<any, AnyField>; focussedField: string; setFocussedField: React.Dispatch<React.SetStateAction<string>>; handleSubmit: (event?: React.SyntheticEvent<HTMLFormElement, Event> | undefined) => Promise<...> | undefined; ... 23 more ...; visited?: { ...; } | undefined; }' is not assignable to parameter of type 'InlineFormRenderChildOptions'.
  Type '{ form: Form<any, AnyField>; focussedField: string; setFocussedField: React.Dispatch<React.SetStateAction<string>>; handleSubmit: (event?: React.SyntheticEvent<HTMLFormElement, Event> | undefined) => Promise<...> | undefined; ... 23 more ...; visited?: { ...; } | undefined; }' is not assignable to type 'Pick<FormRenderProps<any, Partial<any>>, "handleSubmit" | "active" | "dirty" | "dirtyFields" | "dirtyFieldsSinceLastSubmit" | "dirtySinceLastSubmit" | ... 21 more ... | "render">'.
    Types of property 'handleSubmit' are incompatible.
      Type '(event?: SyntheticEvent<HTMLFormElement, Event> | undefined) => Promise<AnyObject | undefined> | undefined' is not assignable to type '(event?: Partial<Pick<SyntheticEvent<Element, Event>, "preventDefault" | "stopPropagation">> | undefined) => Promise<AnyObject | undefined> | undefined'.
        Types of parameters 'event' and 'event' are incompatible.
          Type 'Partial<Pick<SyntheticEvent<Element, Event>, "preventDefault" | "stopPropagation">> | undefined' is not assignable to type 'SyntheticEvent<HTMLFormElement, Event> | undefined'.
            Type 'Partial<Pick<SyntheticEvent<Element, Event>, "preventDefault" | "stopPropagation">>' is missing the following properties from type 'SyntheticEvent<HTMLFormElement, Event>': nativeEvent, currentTarget, target, bubbles, and 9 more.
```

I had to do an extra `//` because `prettier` was moving the `:` around and messing up the ignore.